packages/cms-cli/commands/auth.js

```diff
@@ -20,60 +18,50 @@ const {
   trackCommandUsage,
   addHelpUsageTracking,
 } = require('../lib/usageTracking');
-const { PORTAL_ID, CLIENT_ID, CLIENT_SECRET } = require('../lib/prompts');
+const { promptUser, OAUTH_FLOW } = require('../lib/prompts');
+
+const COMMAND_NAME = 'auth';
+const REQUIRED_SCOPES = ['content'];
 
-const getAuthContext = async () => {
-  const prompt = inquirer.createPromptModule();
-  return prompt([PORTAL_ID, CLIENT_ID, CLIENT_SECRET]);
+const addNewAuthorizedOauthToConfig = async configData => {
```

Comment:
Maybe this should be renamed to "authenticateWithOauth" since it covers not only updating the config but the OAuth flow?

Reply:
Can we also move this out of the `auth` command and into a file that more clearly indicates its shared nature?